src/main/java/com/intellij/rt/debugger/agent/CaptureStorage.java

```diff
@@ -239,8 +347,7 @@ private ExceptionCapturedStack(Throwable exception) {
 
     @Override
     public List<StackTraceElement> getStackTrace() {
-      StackTraceElement[] stackTrace = myException.getStackTrace();
-      return Arrays.asList(stackTrace).subList(1, stackTrace.length);
+      return Arrays.asList(myException.getStackTrace());
```

Comment:
?

Reply:
This one-frame-dropping of `capture` is not needed since we now have a more reliable `fitlerAgentFrames`.